packages/lesswrong/components/linkPreview/HoverPreviewLink.jsx

```diff
@@ -63,9 +62,11 @@ const HoverPreviewLink = ({ innerHTML, href, contentSourceDescription, id }) =>
         const PreviewComponent = parsedUrl.currentRoute.previewComponentName ? Components[parsedUrl.currentRoute.previewComponentName] : null;
         
         if (PreviewComponent) {
-          return <PreviewComponent href={destinationUrl} targetLocation={parsedUrl} innerHTML={innerHTML} id={id}/>
+          return <AnalyticsContext pageElementConetext="linkPreview" href={destinationUrl} hoverPreviewType={parsedUrl.currentRoute.previewComponentName} onsite>
```

Comment:
"pageElementConetext"

Reply:
Also, incorrect indentation

Reply:
In my ideal world, all the props I use would be typed and checked rather than string I can easily mistype.